API.md

```diff
@@ -2106,6 +2106,7 @@ following options:
         - `'error'` - return a Bad Request (400) error response. This is the default value.
         - `'log'` - report the error but continue processing the request.
         - `'ignore'` - take no action and continue processing the request.
+        - `function` - takes a callback with optional params '(request, reply, source, error)'.
```

Comment:
Ok. I see what you mean. Like I said, definitely new to this so some of the phrasing and subtle details often escape me.